test/UnitLoadTorture.cpp

```diff
@@ -23,19 +23,20 @@
 /// Load torture testcase.
 class UnitLoadTorture : public UnitWSD
 {
-    int loadTorture(const std::string& name, const std::string& docName,
-                    const size_t thread_count, const size_t max_jitter_ms);
+    void loadTorture(const std::string& name, const std::string& docName,
+                    const size_t thread_count, const size_t max_jitter_ms = 100);
```

Comment:
Can we set max_jitter to 100 all the time and remove the extra parameter? I see it is called with 75 below, but I don't think 75 vs 100 would make much difference.

Reply:
Quite possibly; but there was probably some thought here.